src/_igraph/graphobject.c

```diff
@@ -18693,6 +18835,46 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "@return: a random walk that starts from the given vertex and has at most\n"
    "  the given length (shorter if the random walk got stuck).\n"
   },
+  
+  /****************/
+  /* OTHER METHODS */
+  /****************/{
```

Comment:
Let's add this to the `COMMUNITIES AND DECOMPOSITION` section instead of introducing a new section.